shared/scss/abstracts/_variables.scss

```diff
@@ -0,0 +1,9 @@
+/*------------------------------------*\
+  #FONT
+\*------------------------------------*/
+
+// font weight
+$font-weight-regular: 400;
+$font-weight-medium: 500;
+$font-weight-semibold: 600;
+$font-weight-bold: 700;
```

Comment:
The same as before, please remove the comment in line 5, and use the camelCase naming convention